### DISCUSSION TITLE
Custom Namespace in Laravel 11: Issue with Artisan Commands

### DISCUSSION THREAD

**Discussion by dwghihanw** [laravel/framework#51285] [Q&A]
Hi everyone,

I've been working with Laravel 11 and decided to change the default namespace ("App") to a custom one ("Aggp"). I made the necessary adjustments in the composer file and updated all existing files accordingly. The application is functioning smoothly with the custom namespace.

However, I've encountered an issue where the codes generated through artisan commands still use the default namespace. I've searched extensively but couldn't find a solution to change this setting.

I'd greatly appreciate any help or advice on how to resolve this matter.

Best regards,
Gihan Wijerathna

**Comment by joy2362** [comment]
when create controller or something that inside app folder you can add custom namespace like
 `php artisan make:controller CustomNamespace/YourControllerName --namespace=CustomNamespace`
